diff -Nur -x '*.orig' -x '*.rej' smeserver-spamassassin-1.4.0/root/etc/e-smith/templates/etc/mail/spamassassin/local.cf/10paths mezzanine_patched_smeserver-spamassassin-1.4.0/root/etc/e-smith/templates/etc/mail/spamassassin/local.cf/10paths
--- smeserver-spamassassin-1.4.0/root/etc/e-smith/templates/etc/mail/spamassassin/local.cf/10paths	2006-06-09 16:49:12.953663248 +1000
+++ mezzanine_patched_smeserver-spamassassin-1.4.0/root/etc/e-smith/templates/etc/mail/spamassassin/local.cf/10paths	2006-06-09 16:48:42.750916859 +1000
@@ -1,2 +1,4 @@
 bayes_path /var/spool/spamd/.spamassassin/bayes
 bayes_file_mode 750
+auto_whitelist_path /var/spool/spamd/.spamassassin/auto-whitelist
+auto_whitelist_file_mode 750
