diff -Nur -x '*.orig' -x '*.rej' smeserver-spamassassin-1.4.0/root/etc/e-smith/templates-user/.qmail/15sortspam mezzanine_patched_smeserver-spamassassin-1.4.0/root/etc/e-smith/templates-user/.qmail/15sortspam
--- smeserver-spamassassin-1.4.0/root/etc/e-smith/templates-user/.qmail/15sortspam	2005-09-24 07:25:26.000000000 +1000
+++ mezzanine_patched_smeserver-spamassassin-1.4.0/root/etc/e-smith/templates-user/.qmail/15sortspam	2006-07-13 14:35:59.067835832 +1000
@@ -11,5 +11,5 @@
 
     return "# sortspam disabled" unless ($do_sortspam eq "enabled");
 
-    return qq(| condredirect $USERNAME-junkmail /bin/grep -q -i 'X-Spam-Status: Yes');
+    return qq(| condredirect $USERNAME-junkmail /bin/grep -q -i '^X-Spam-Status: Yes');
 }
