diff -Nur -x '*.orig' -x '*.rej' smeserver-spamassassin-1.3.0/root/var/service/spamd/run mezzanine_patched_smeserver-spamassassin-1.3.0/root/var/service/spamd/run
--- smeserver-spamassassin-1.3.0/root/var/service/spamd/run	2005-09-24 07:25:26.000000000 +1000
+++ mezzanine_patched_smeserver-spamassassin-1.3.0/root/var/service/spamd/run	2005-11-03 03:12:35.242805180 +1100
@@ -21,4 +21,8 @@
 #----------------------------------------------------------------------
 exec 2>&1
 
-exec /usr/bin/spamd -u spamd --syslog=stderr
+SPAMD_STATUS=$(/sbin/e-smith/db configuration getprop spamassassin status)
+
+if [ "$SPAMD_STATUS" == "enabled" ]; then
+  exec /usr/bin/spamd -u spamd --syslog=stderr
+fi
