diff -Nur -x '*.orig' -x '*.rej' smeserver-spamassassin-1.4.0/root/etc/e-smith/templates/etc/mail/spamassassin/local.cf/10required_hits mezzanine_patched_smeserver-spamassassin-1.4.0/root/etc/e-smith/templates/etc/mail/spamassassin/local.cf/10required_hits
--- smeserver-spamassassin-1.4.0/root/etc/e-smith/templates/etc/mail/spamassassin/local.cf/10required_hits	2005-09-24 07:25:26.000000000 +1000
+++ mezzanine_patched_smeserver-spamassassin-1.4.0/root/etc/e-smith/templates/etc/mail/spamassassin/local.cf/10required_hits	1970-01-01 10:00:00.000000000 +1000
@@ -1,16 +0,0 @@
-{ 
-    my $sens = $spamassassin{Sensitivity} || 'medium';
-
-    my %sens2hits =	( 
-			'veryhigh'	=> 2,
-			'high'		=> 3,
-			'medium' 	=> 5,
-			'low'	 	=> 7,
-			'verylow'	=> 9,
-			);
-
-    my $hits = ($sens eq 'custom') ? $spamassassin{TagLevel}
-				   : $sens2hits{$sens};
-
-    return "required_hits $hits";
-}
diff -Nur -x '*.orig' -x '*.rej' smeserver-spamassassin-1.4.0/root/etc/e-smith/templates/etc/mail/spamassassin/local.cf/10required_score mezzanine_patched_smeserver-spamassassin-1.4.0/root/etc/e-smith/templates/etc/mail/spamassassin/local.cf/10required_score
--- smeserver-spamassassin-1.4.0/root/etc/e-smith/templates/etc/mail/spamassassin/local.cf/10required_score	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-spamassassin-1.4.0/root/etc/e-smith/templates/etc/mail/spamassassin/local.cf/10required_score	2008-03-09 09:07:00.000000000 +1100
@@ -0,0 +1,16 @@
+{ 
+    my $sens = $spamassassin{Sensitivity} || 'medium';
+
+    my %sens2hits =	( 
+			'veryhigh'	=> 2,
+			'high'		=> 3,
+			'medium' 	=> 5,
+			'low'	 	=> 7,
+			'verylow'	=> 9,
+			);
+
+    my $hits = ($sens eq 'custom') ? $spamassassin{TagLevel}
+				   : $sens2hits{$sens};
+
+    return "required_score $hits";
+}
